kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/KafkaUtils.java

```diff
@@ -202,6 +203,9 @@ public static boolean restartBroker(String deployNamespace, String clusterName)
                 break;
             }
         }
+        if (podName.isEmpty() || podName.isBlank()) {
+            throw new KubeClusterException(new Throwable("Kafka cluster name not found!"));
```

Comment:
Creating a throwable is weird.  Just add another ctor to KubeClusterException that accepts a String.

Reply:
done